lib/ht/search_client/test.rb

```diff
@@ -4,6 +4,10 @@ module Ht::SearchClient::Test
     def stub_request(params = {})
       new(params).stub_request
     end
+
+    def stub_post_request(params = {})
```

Comment:
Can't we include an option on params to have the http method and not have additional methods OR even figure it out on its own ? i.e. the testing class handles this for you ? Or you brainstormed this and you think it's more simple to be aware this endpoint will always require a POST ?